samples/azure/azure-service-fabric-routing/Core_6/Shared/ReceiverSideDistribution/PartitionMappingFailedException.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Shared
+{
+    [Serializable]
+    public class PartitionMappingFailedException : Exception
```

Comment:
unless this crosses appdomain boundaries the extra overloads and `[Serializable]` is redundant